server/controllers/auth.rb

```diff
@@ -15,13 +15,17 @@ class AuthController < ApplicationController
         .to_json(root: true, :only => [:username, :id])
     else
       requires_role! :admin
-      User.to_json(root: true)
+      User.to_json(root: true, exclude: :avatar)
```

Comment:
_Random Note:_ So now anywhere we return a user object we will have to both exclude the avatar and also exclude the password hash / salt.